vapi/libecal-1.2.vapi

```diff
@@ -307,7 +307,7 @@ namespace ECal {
 		public weak string cn;
 		public weak string language;
 	}
-	[CCode (cheader_filename = "libecal/libecal.h", free_function = "e_cal_component_free_datetime")]
+	[CCode (cheader_filename = "libecal/libecal.h", free_function = "e_cal_component_free_datetime", has_copy_function = false, has_destroy_function = false)]
```

Comment:
See above

Reply:
Vala compiler tries to call copy and destroy functions for this structure and it doesn't have those. This pull request doesn't need those changes anymore but this commit fixes the incorrect VAPI for this structure. I can discard this change for this pull request but a VAPI change may be required in the future.